src/Components/Form-Settings/FormSettings.component.js

```diff
@@ -118,6 +133,27 @@ export default class FormSettings extends Component {
         this.setState({ tempSelectedColumns });
     };
 
+    applyChangesToAll = async () => {
```

Comment:
No need to write same method once again just for single parameter. Please use 'applyChanges'